### PR TITLE
hamlib 3.3

### PR DESCRIPTION
**Pull request by sirhc808**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/hamlib.rb [hunk 1]
```diff
@@ -1,18 +1,8 @@
 class Hamlib < Formula
   desc "Ham radio control libraries"
   homepage "https://hamlib.sourceforge.io/"
-  url "https://src.fedoraproject.org/repo/pkgs/hamlib/hamlib-1.2.15.3.tar.gz/3cad8987e995a00e5e9d360e2be0eb43/hamlib-1.2.15.3.tar.gz"
-  sha256 "a2ca4549e4fd99d6e5600e354ebcb57502611aa63c6921c1b8a825289833f75e"
-
-  bottle do
```
Comment (SMillerDev) [inline Formula/hamlib.rb]: please don't remove the bottle block

**Comment by sirhc808** [comment]
Thanks Sean, new to Homebrew, first commit.  Do I need to change the bottle sha256 values or just leave as is?  I was going for minimal w my commit.  Thanks!

File: Formula/hamlib.rb (hunk 1 above)
Comment (sirhc808) [reply Formula/hamlib.rb]: Thanks Sean, new to Homebrew, first commit.  Do I need to change the bottle sha256 values or just leave as is?  I was going for minimal w my commit.  Thanks!

Comment (fxcoudert) [reply Formula/hamlib.rb]: Just leave it as is, and our CI and merging procedure will deal with it automatically!

**Comment by fxcoudert** [comment]
Please squash your 2 commits into a single one, with commit message `hamlib 3.3`

**Comment by sirhc808** [comment]
Thanks!  I started over with a single commit and submitted PR 

	https://github.com/Homebrew/homebrew-core/pull/41630 <https://github.com/Homebrew/homebrew-core/pull/41630>

Many thanks!

All the best,
-Chris

**Pull request closed without merging**